prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -106,6 +106,9 @@ def _describe_replication_groups(self, regional_client):
                             auto_minor_version_upgrade=repl_group.get(
                                 "AutoMinorVersionUpgrade", False
                             ),
+                            automatic_failover=repl_group.get(
+                                "AutomaticFailover", None
```

Comment:
We always set the non-compliant value as the default.
```suggestion
                                "AutomaticFailover", "disabled"
```